proto/grp.proto

```diff
@@ -18,3 +20,22 @@ message Group {
   // True if this group has linked a Github token.
   bool github_linked = 4;
 }
+
+message CreateGroupRequest {
+  string name = 1;
+  string owned_domain = 2;
```

Comment:
think we should either have owned_domain or the auto_populate_from_owned_domain bool below (same in update), but both opens up a lot of weird edge cases
 - what happens if the owned_domain is different than the users email address?
 - what happens if owned_domain is set, but auto_populate_from_owned_domain is not set?

i lean towards just having the bool, and populating owned_domain from the user's email address depending on whether or not it's set

Reply:
Makes sense, definitely simpler to just have a bool. Done